src/components/views/SuggestedViews/ViewCard.tsx

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs';
+import { ReactEventHandler } from 'react';
+import { ZetkinView } from 'types/zetkin';
+import { Card, CardActionArea, Fade, Grid, Typography } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    action: {
+        padding: theme.spacing(2),
+    },
+    content: {
+        minHeight: 200,
+    },
+}));
+
+type ViewCardProps = {
+    onClick: ReactEventHandler;
+    view: ZetkinView;
+}
+
+const ViewCard: React.FunctionComponent<ViewCardProps> = ({ onClick,  view }) => {
+    const classes = useStyles();
+
+    if (!view) return <div />;
+    return (
+        <Fade in>
+            <Card>
+                <CardActionArea className={ classes.action } onClick={ onClick } value={ view.id }>
+                    <Grid className={ classes.content } container direction="column" justifyContent="space-between">
```

Comment:
I wouldn't mind if these lines were shorter. Maybe 80 as a soft limit is better (as you suggested at some point)?

Reply:
Why I want prettier in every repo.... ;-)